packages/mui-material/src/Select/SelectInput.js

```diff
@@ -324,6 +328,24 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
     }
   };
 
+  const handleItemKeyDown = (child) => (event) => {
+    if (event.key === ' ') {
+      const newValue = child.props.value;
+      if (newValue !== value) {
+        setValueState(newValue);
+
+        if (onChange) {
+          clonedOnChange(event, newValue, child);
+        }
+      }
+      update(false, event);
+    }
+
+    if (child.props.onKeyDown) {
+      child.props.onKeyDown(event);
+    }
```

Comment:
```suggestion
    child?.props?.onKeyDown?.(event);
```

To be safer & simpler :)